img/user.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
I don't see where those svg are used? :)

Reply:
yes it is used, see here
![image](https://user-images.githubusercontent.com/9105243/74089215-41a7e880-4a9f-11ea-9046-adb93323f39d.png)